plugin/pkg/scheduler/algorithm/listers.go

```diff
@@ -184,3 +185,42 @@ func (f FakeReplicaSetLister) GetPodReplicaSets(pod *v1.Pod) (rss []*extensions.
 
 	return
 }
+
+// StatefulSetLister interface represents anything that can produce a list of StatefulSet; the list is consumed by a scheduler.
+type StatefulSetLister interface {
+	// Gets the StatefulSet for the given pod.
+	GetPodStatefulSets(*v1.Pod) ([]*apps.StatefulSet, error)
```

Comment:
@timothysc 
I am not sure what you mean by "hit the wire". This method of the interface is called by the actual scheduler code.
I do not know about the informer part. It must be something that I have missed completely. Is there any pointer?